packages/node_modules/@webex/spark-widget-base/src/enhancers/withBrowserGlobals.js

```diff
@@ -166,11 +170,11 @@ function setupSparkWidgetNamespace() {
  */
 export default function withBrowserGlobals({name}) {
   // Inject widget into browser globals
-  setupSparkWidgetNamespace();
+  setupWidgetNamespace();
   const fnName = `${name}Widget`;
 
   return (BaseComponent) => {
-    function renderSparkWidget(options) {
```

Comment:
Not sure if you wanted to include `Webex` here, but looks good otherwise.

Reply:
I figured removing the product name from the function name would make it a bit more future proof. It doesn't need to be specific since there's only one type of widget in this codebase.